CONTRIBUTING.md

```diff
@@ -45,6 +45,41 @@ Before you can start contributing to the CyberArk Secrets Provider for Kubernete
     
     b. For detailed setup instructions for OSS, see [CyberArk Secrets Provider for Kubernetes for OSS](https://docs.conjur.org/Latest/en/Content/Integrations/Kubernetes_deployApplicationsConjur-k8s-Secrets.htm).
     
+### Deploy a Local Dev Environment
+
+You can now deploy a local development environment for Kubernetes using [Docker Desktop](https://www.docker.com/products/docker-desktop). Docker Desktop provides a convenient way to deploy and develop from your machine against a cluster deployed locally. 
+To enable this, perform the following:
+
+1. Navigate to `bootstrap.env` and uncomment the `Local DEV Env` section, ensuring that `DEV=true`
+
+1. Run `./bin/start --dev --gke`, appending `--oss` or `--dap` according to the environment that needs to be deployed 
```

Comment:
i would add that after the script is run you can run `kubectl get pods` to see the pod of the secerts-provider. and run `kubectl logs <pod-name> -c cyberark-secrets-provider` to view its logs

Reply:
also - how do i stop the env?

Reply:
as in delete env? adding clean up section in contributing docs to explain how to delete deployments / resources / etc